third_party/wasm-polyfill/Makefile

```diff
@@ -0,0 +1,41 @@
+# Patches welcome to actually make this portable
```

Comment:
I'd prefer we would nuke this and never build via a Makefile, because of the portability issue. Let's do either a python script or a CMake project, either of which are much more portable?